modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContextImpl.java

```diff
@@ -183,18 +183,22 @@ public ObjectEntryDisplayContextImpl(
 	}
 
 	public String getAPIURL() throws PortalException {
-		ObjectRelationship objectRelationship =
-			_objectRelationshipLocalService.fetchObjectRelationship(
-				ParamUtil.getLong(
-					_objectRequestHelper.getRequest(), "objectRelationshipId"));
-
 		String externalReferenceCode = null;
 
 		if (_objectEntry != null) {
 			externalReferenceCode = _objectEntry.getExternalReferenceCode();
 		}
 
+		ObjectRelationship objectRelationship =
+			_objectRelationshipLocalService.fetchObjectRelationship(
+				ParamUtil.getLong(
+					_objectRequestHelper.getRequest(), "objectRelationshipId"));
```

Comment:
Can you move it back this variable to where it was?

I think the order is by usage:

ObjectRelationship **objectRelationship** =
			_objectRelationshipLocalService.fetchObjectRelationship(
				ParamUtil.getLong(
					_objectRequestHelper.getRequest(), "objectRelationshipId"));

String externalReferenceCode = null;

if (_objectEntry != null) {
	externalReferenceCode = _objectEntry.getExternalReferenceCode();
}

if ((**objectRelationship** == null) || !objectRelationship.isEdge()) {
	return _getAPIURL(externalReferenceCode, getObjectDefinition1());
}